internal/dinosql/parser.go

```diff
@@ -958,7 +987,8 @@ type paramRef struct {
 type paramSearch struct {
 	parent   nodes.Node
 	rangeVar *nodes.RangeVar
-	refs     map[int]paramRef
+	refs     *[]paramRef
```

Comment:
So I tried to make `Visit` take a pointer receiver: `(p *paramSearch)`, rather than doing a pointer to a slice here. This broke the `DeleteVenue` query parsing somehow. The parent for the second parameter because a `FunctionCall` instead of an `Expr`. I'm a compilers noob so I decided to not dive down that rabbit hole.